Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -607,6 +631,20 @@ struct ActivityStreamTracker {
     let eventsTracker: PingCentreClient
     let sessionsTracker: PingCentreClient
 
+    func reportBadState(badState: ASPingBadStateEvent, source: ASPingSource) {
+        var eventPing: [String: Any] = [
+            "event": badState.rawValue,
+            "page": "NEW_TAB",
+            "source": source.rawValue,
+            "app_version": AppInfo.appVersion,
```

Comment:
To avoid duplicating eventPing creation. Why dont we move the basic data we use in each ping (app version/build) etc into the actual sendPing method. If we cant do this (because looks like PingCenter isnt linked to Shared) maybe create a base ping in AS itself.